src/components/Video/index_old.jsx

```diff
@@ -0,0 +1,793 @@
+import React, { useCallback, useEffect, useRef, useState } from "react";
```

Comment:
@raghavdhingra , do we need this index_old.jsx?

Reply:
If index.jsx file is tested thoroughly then we can delete the old jsx file @sansyrox